src/app/shared/components/toast/toast.component.ts

```diff
@@ -45,11 +47,9 @@ export class ToastComponent implements OnInit, OnDestroy {
   }
 
   copyToClipboard(): void {
-    const text = document.querySelector('#detailedErrorMessage')!;
-    navigator.clipboard.writeText(text.innerHTML).then(() => {
-      const button = document.querySelector('#CopyToClipboard')!;
-      button.innerHTML = 'Copied!';
-      button.setAttribute('style', 'background-color: #23c6c8 !important');
+    navigator.clipboard.writeText(this.detailedErrorMessageElement.nativeElement.innerHTML).then(() => {
```

Comment:
Does this work on Firefox? I know Firefox doesnt have the new Clipboard API implemented yet and the old one is already deprecated.
If it works then fine, if it doesnt then use the deprecated clipboard way